packages/geolocator/example/lib/main.dart

```diff
@@ -0,0 +1,414 @@
+            break;
+          case 2:
+            _requestTemporaryFullAccuracy();
+            break;
+          case 3:
+            _openAppSettings();
+            break;
+          case 4:
+            _openLocationSettings();
+            break;
+          case 5:
+            setState(_positionItems.clear);
+            break;
+          default:
+            break;
+        }
+      },
+      itemBuilder: (context) => [
+        if (Platform.isIOS)
+          const PopupMenuItem(
+            child: Text("Get Location Accuracy"),
+            value: 1,
+          ),
+        if (Platform.isIOS)
+          const PopupMenuItem(
+            child: Text("Request Temporary Full Accuracy"),
+            value: 2,
+          ),
+        const PopupMenuItem(
+          child: Text("Open App Settings"),
+          value: 3,
+        ),
+        if (Platform.isAndroid)
+          const PopupMenuItem(
+            child: Text("Open Location Settings"),
+            value: 4,
+          ),
+        const PopupMenuItem(
+          child: Text("Clear"),
+          value: 5,
+        ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    const sizedBox = SizedBox(
+      height: 10,
+    );
+
+    return BaseflowPluginExample(
+        pluginName: 'Geolocator',
+        githubURL: 'https://github.com/Baseflow/flutter-geolocator',
+        pubDevURL: 'https://pub.dev/packages/geolocator',
+        appBarActions: [
+          _createActions()
+        ],
+        pages: [
+          ExamplePage(
+            Icons.location_on,
+            (context) => Scaffold(
+              backgroundColor: Theme.of(context).backgroundColor,
+              body: ListView.builder(
+                itemCount: _positionItems.length,
+                itemBuilder: (context, index) {
+                  final positionItem = _positionItems[index];
+
+                  if (positionItem.type == _PositionItemType.log) {
+                    return ListTile(
+                      title: Text(positionItem.displayValue,
+                          textAlign: TextAlign.center,
+                          style: const TextStyle(
+                            color: Colors.white,
+                            fontWeight: FontWeight.bold,
+                          )),
+                    );
+                  } else {
+                    return Card(
+                      child: ListTile(
+                        tileColor: themeMaterialColor,
+                        title: Text(
+                          positionItem.displayValue,
+                          style: const TextStyle(color: Colors.white),
+                        ),
+                      ),
+                    );
+                  }
+                },
+              ),
+              floatingActionButton: Column(
+                crossAxisAlignment: CrossAxisAlignment.end,
+                mainAxisAlignment: MainAxisAlignment.end,
+                children: [
```

Comment:
As far as I know, some examples of plug-in packages have UI overflow problems on wearable(or the layout is not suitable for circular displays).
I think we need to think about whether fix up UI related problems of example apps or leave them as is.

Reply:
You've specified that this plugin only supports the watch profile. Thus this app shouldn't have UI problems at least on watch devices. I don't care example apps of other plugins that support multiple profiles.

Reply:
For the overflowed floating button, you may just remove `sizedBox` widgets from `floatingActionButton`'s children.

For the actions button, this change will do:
```dart
appBarActions: [
  Padding(
    padding: const EdgeInsets.only(right: 50),
    child: _createActions(),
  ),
],
```

Reply:
Some people say that please leave upstream code as-is and others say that please fix it in some cases. 
this example is also from upstream. which one is preferable? I feel our policies are inconsistent now :(  

IMHO, if we start to fix examples to some purpose such as to fit UI, our default stance on upstream code is to leave the original code, which I think should be reconsidered.

Just to tell you what I think, the location API supports all profile except TV.  this can be run on Tizen emulator for each profile and it works fine. also, commercial devices can be added over time, and I think our basic stance is supporting multi profile. So it's not bad to leave it as it is.

the reason why I specified the wearable only in README is because I think that our commonsense is that we only specified some device among commercial products.

Reply:
> For the actions button, this change will do:

it works fine. after this conversation I will update example or not, including remove isAndroid

Reply:
I always kept saying "change the original code as necessary if you need". We can't **always** use the original code as it is although it's **preferable** to not change it as much as possible for maintenance reason. As far as I remember there are already many examples and tests that differ from the original code.

> the reason why I specified the wearable only in README is because I think that our commonsense is that we only specified some device among commercial products.

There are only two officially supported profiles for our plugins: watch and TV. Some plugins may implicitly support other profiles, and you may test the plugins on those devices, but they're basically out of scope.

> Just to tell you what I think, the location API supports all profile except TV. this can be run on Tizen emulator for each profile and it works fine. also, commercial devices can be added over time, and I think our basic stance is supporting multi profile. So it's not bad to leave it as it is.

It's bad. Some part of this example app can't literally function because the actions button is outside the screen. At least you should modify the actions button part.